joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

yb_arm_joint_0_position_controller:
  type: effort_controllers/JointPositionController
  joint: yb_arm_joint_0
  pid: {p: 1000.0, i: 0.01, d: 10}

yb_arm_joint_1_position_controller:
  type: effort_controllers/JointPositionController
  joint: yb_arm_joint_1
  pid: {p: 1000.0, i: 0.01, d: 10}

yb_arm_joint_2_position_controller:
  type: effort_controllers/JointPositionController
  joint: yb_arm_joint_2
  pid: {p: 1000.0, i: 0.01, d: 10}

yb_arm_joint_3_position_controller:
  type: effort_controllers/JointPositionController
  joint: yb_arm_joint_3
  pid: {p: 1000.0, i: 0.01, d: 10}

yb_arm_joint_4_position_controller:
  type: effort_controllers/JointPositionController
  joint: yb_arm_joint_4
  pid: {p: 1000.0, i: 0.01, d: 10}

yb_arm_joint_5_position_controller:
  type: effort_controllers/JointPositionController
  joint: yb_arm_joint_5
  pid: {p: 1000.0, i: 0.01, d: 10}